docs/docs-generator.js

```diff
@@ -23,7 +23,7 @@ if (program.file) {
     {file:'lib/sequelize.js', output: 'sequelize'},
     {file:'lib/instance.js', output: 'instance'},
     {file:'lib/model.js', output: 'model'},
-    {file:'lib/querying.js', output: 'querying'},
```

Comment:
Yeah but that file is not generated by the docs-generator but manually authored right?